mne/decoding/tests/test_time_gen.py

```diff
@@ -424,4 +424,22 @@ def test_decoding_time():
     tg = TimeDecoding(clf=clf, cv=cv)
     tg.fit(epochs, y=y)
 
+
+@requires_sklearn
+def test_string_scorer():
+    """Test whether passing string as scorer works as intended"""
+    epochs = make_epochs()
+    epochs.crop(epochs.times[0], epochs.times[2])
+    td_1 = TimeDecoding(scorer='accuracy')
+    td_1.fit(epochs)
+    score_1 = td_1.score(epochs)
+
+    td_2 = TimeDecoding()
+    td_2.fit(epochs)
+    score_2 = td_2.score(epochs)
+    assert_array_equal(score_1, score_2)
+
+    td_1.scorer = 'accuracies'
+    assert_raises(KeyError, td_1.score, epochs)
```

Comment:
how long does this take to run?

can you test this without introducing a new test?